Casks/evernote.rb

```diff
@@ -28,10 +28,11 @@
   livecheck do
     url "https://evernote.s3.amazonaws.com/boron/mac/public/latest-mac.yml"
     strategy :electron_builder do |yml|
```

Comment:
```suggestion
    regex(/Evernote[._-](\d+(?:\.\d+)+)-mac-ddl-ga-(\d+(?:\.\d+)*)-([0-9a-f]+)\.dmg/i)
    strategy :electron_builder do |yaml, regex|
```

Unfortunately I can't create one suggestion that spans the entire `livecheck` block (since GitHub doesn't allow a suggestion for removed lines), so I'm having to split this up.

`livecheck` block regexes should be case-insensitive by default. The suggestion above cleans the regex up a bit, as we probably won't benefit from the excessive use of `[._-]` in the `-mac-ddl-ga-` part of the filename.

Additionally, the regex shouldn't be defined inside the `strategy` block. The `#regex` call should happen outside of the `strategy` block and simply get passed in.

Reply:
thanks, i wanted to make minimal changes to original regex that someone else wrote, but these are good suggestions